proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -72,6 +74,8 @@ message ForwardMsg {
     // for this one. If the client does not have the referenced message
     // in its cache, it can retrieve it from the server.
     string ref_hash = 11;
+
+    // NEXT ID: 13
```

Comment:
It's cool having a comment for the next ID but looks like we do this when we alphabetize everything. Could we either alphabetize or number sequentially the whole thing?